stable/appmesh-manager/upgrade/pre_upgrade_check.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+check_old_crds() {
+
+    vs=$(kubectl get crd virtualservices.appmesh.k8s.aws --ignore-not-found -o json | jq -r '.spec.versions[]? | select(.? | .name == "v1beta1")')
+    vn=$(kubectl get crd virtualnodes.appmesh.k8s.aws --ignore-not-found -o json | jq -r '.spec.versions[]? | select(.? | .name == "v1beta1")')
+    ms=$(kubectl get crd meshes.appmesh.k8s.aws --ignore-not-found -o json | jq -r '.spec.versions[]? | select(.? | .name == "v1beta1")')
+
+    if [[ -z $vs && -z $vn && -z $ms  ]]; then
+        echo "App Mesh CRD check: PASSED!"
+        return 0
+    else
+       echo "App Mesh CRD check: FAILED -- v1beta1 CRDs are still installed"
+       return 1
+    fi
+
+}
+
+check_controller_version() {
+    currentver=$(kubectl get deployment -n appmesh-system appmesh-controller --ignore-not-found -o json | jq -r ".spec.template.spec.containers[].image" | cut -f2 -d ':')
+    requiredver="v1.0.0"
+
```

Comment:
i believe we renamed the deployment to appmesh-manager, should this message change too?

Reply:
I think it'd be better to keep it as is as some users may go looking for appmesh-manager in the older version. Also, we may have to adjust the name based on the earlier discussion

Reply:
yeah, let's see whether we keep appmesh-controller or manager :D